test/dotnet-build.Tests/IncrementalTestBase.cs

```diff
@@ -12,6 +12,9 @@ namespace Microsoft.DotNet.Tools.Builder.Tests
 {
     public class IncrementalTestBase : TestBase
     {
+        protected readonly string _libraryFrameworkFullName = ".NETStandard,Version=v1.5";
+        protected readonly string _appFrameworkFullName = ".NETCoreApp,Version=v1.0";
```

Comment:
The meaningful difference is that every instance of IncrementalTestBase will get a reference to this string, which bloats the object. And every time you access it, it goes back to the reference to get the value. When a member is marked `const` each object won't get its own reference. Instead the constant will get in-lined wherever it is used.

Obviously this isn't a huge deal in tests. In general .NET programming it is better to use `const` when possible for non-public/protected constants. When you have a public/protected constant that could possibly change in the future, then you should use readonly instead of const.